docs/docs/README.md

```diff
@@ -1,8 +1,8 @@
 # Atlassian Data Center Helm Charts
 
 [![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/atlassian-data-center)](https://artifacthub.io/packages/search?repo=atlassian-data-center)
-[![Atlassian license](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=flat-square)](https://github.com/atlassian-labs/data-center-helm-charts/blob/master/CONTRIBUTING.md) 
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/atlassian-labs/data-center-helm-charts/blob/master/CONTRIBUTING.md) 
+[![Atlassian license](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=flat-square)](https://github.com/atlassian-labs/data-center-helm-charts/blob/main/CONTRIBUTING.md) 
```

Comment:
Can you also change `CONTRIBUTING.md` into `LICENSE`. I think this is just an oversight.